safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -294,6 +444,7 @@ def prepare(self):
                 from the code.
         :rtype: (int, m.Message)
         """
+        self._provenance_ready = False
         if len(self._exposures) < 2:  # 2 layers minimum.
             message = generate_input_error_message(
                 tr('No exposure layer provided'),
```

Comment:
Perhaps clearer error messagem like, "Needs at least 2 exposure layers"

Reply:
You see only the title, the description is the next line explaining they need 2 layers. I can still improve it

Reply:
Ah yes, sorry. Sometime github cut it in the middle of something. Thanks